src/main/resources/templates/recipe.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
I added a recipe page template or what one could look like for your recipes when they are displayed on the webpage.